cpp/src/parquet/file_reader.h

```diff
@@ -22,6 +22,7 @@
 #include <string>
 #include <vector>
 
+#include "parquet/column_reader.h"
```

Comment:
This should after `#include "arrow/util/type_fwd.h"` ?

Reply:
Done.